regolith/helpers/u_todohelper.py

```diff
@@ -96,51 +103,81 @@ def db_updater(self):
                     "in the command line")
                 return
         filterid = {'_id': rc.assigned_to}
-        person = rc.client.find_one(rc.database, rc.coll, filterid)
+        if rc.running_index:
+            index = 1
+            index_finished = -1
+            for i in range(0, len(rc.databases)):
+                db_name = rc.databases[i]["name"]
+                person_idx = rc.client.find_one(db_name, rc.coll, filterid)
+                todolist_idx = person_idx.get("todos", [])
+                if len(todolist_idx) == 0:
+                    continue
+                else:
+                    for todo in todolist_idx:
+                        if todo.get('status') == "started":
+                            todo["running_index"] = index
+                            index += 1
```

Comment:
First, give each task a temporary running_index that will be used to identify each task later